### ISSUE TITLE
Add anvaka/map-of-github to the map

### ISSUE THREAD

**Issue by suhr** [anvaka/map-of-github#8]

**Comment by anvaka** [comment]
Good call. Looks like the project got enough stars to find potential neighbors. I'll let it collect more stars and will update it eventually (hopefully in a month)
